lms/static/js/factories/cohorts_factory.js

```diff
@@ -1,33 +1,40 @@
 ;(function (define, undefined) {
```

Comment:
I think a better path for this file will be groups/views/cohorts_dashboard_factory.js.